src/components/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from "react";
+// custom css import
+import "./Navbar.css";
+// icons import
+import HomeIcon from "@mui/icons-material/Home";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import AddCircleOutlineIcon from "@mui/icons-material/AddCircleOutline";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+
+function Navbar() {
+  return (
+    <div className="nav-bar">
+      <ul>
+        <li>
+          <a href="">
```

Comment:
instead of <a> anchor tags use Link components from [material ui](https://mui.com/material-ui/react-link/) and provide links to correct routes... for example home is "/", newPost component can be extracted from app.jsx and rendered as an element directly [documentation link](https://mui.com/material-ui/api/link/) , favourities and profile can be loaded directly... don't worry if they're not working as expected for now. Just render them.